### DISCUSSION TITLE
Help remove those panel

### DISCUSSION THREAD

**Discussion by kakachiex2** [Comfy-Org/ComfyUI#4413] [Q&A]
Can anyone know from what extension are this big panel I don't find a way to remove it 
![Screenshot 2024-08-16 124859](https://github.com/user-attachments/assets/ca0e8a7b-95fa-4413-99a7-c49c2d14e0d2)

**Comment by kakachiex2** [comment]
This panel is eating my space
![Screenshot 2024-08-16 124859](https://github.com/user-attachments/assets/d5de7675-33d1-42b1-afaf-b317bdb0f5ac)

**Comment by JorgeR81** [comment]
I remember seen similar a image.
https://github.com/comfyanonymous/ComfyUI/issues/3894#issuecomment-2198472968


Try disabling: 
- ComfyUI-3D-Pack
- ComfyUI-N-Sidebar

https://github.com/comfyanonymous/ComfyUI/issues/3894#issuecomment-2198589767
